keycloak/keycloakify/Dockerfile

```diff
@@ -11,14 +11,10 @@ RUN mvn --version
 
 WORKDIR /app
 RUN wget  https://github.com/eosc-kc/keycloak-orcid/releases/download/${KEYCLOAK_ORCID_VERSION}/keycloak-orcid-${KEYCLOAK_ORCID_VERSION}.jar
-COPY package.json yarn.lock .yarnrc.yml ./
-COPY .yarn/releases .yarn/releases
-RUN corepack enable
-RUN corepack install
-RUN yarn install --immutable && \
-    yarn cache clean
-COPY . .
```

Comment:
N.B. you are no longer copying most of the files if I understand it (I know you're debugging something else atm, but for reference)

Reply:
Oh damn, thanks!